src/twisted/internet/test/test_inlinecb.py

```diff
@@ -197,6 +907,42 @@ def calling():
         self.assertIn("Error Marker", tb)
         self.assertIn("in erroring", f.getTraceback())
 
+    def test_reraiseTracebacksFromDeferred(self) -> None:
+        """
+        L{defer.inlineCallbacks} that receives tracebacks from a regular Deferred and
+        re-raise tracebacks into their deferred should not lose their tracebacks.
+        """
+        f = getDivisionFailure("msg")
+        d: Deferred[None] = Deferred()
+        try:
+            f.raiseException()
+        except BaseException:
+            d.errback()
+
+        def ic(d: object) -> Generator[Any, Any, None]:
+            """
+            This is never called.
+            It is only used as the decorated function.
+            The resulting function is never called in this test.
+            This is used to make sure that if we wrap
+            an already failed deferred, inlineCallbacks
+            will not add any extra traceback frames.
+            """
+            yield d
```

Comment:
If we know that a line is never called, is best to also let our coverage tools know about this :)

```suggestion
            yield d  # pragma: no cover
```

Reply:
I see that this code was jus moved... but is better to add the coverage pragma info now.

Thanks.

Reply:
Applied, thanks.